Core/Src/DynamicControls/PID.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * PID.c
+ *
+ *  Created on: Apr 14, 2020
+ *      Author: renanmoreira
+ */
+
+#include <DynamicControls/PID.h>
```

Comment:
```suggestion
#include "DynamicControls/PID.h"
```
usa <> só pra includes de bibliotecas standard, tipo <stdio.h>, etc